docs/fundamentals/code-analysis/quality-rules/ca1812.md

```diff
@@ -62,7 +62,7 @@ It is safe to suppress a warning from this rule. We recommend that you suppress
 
 - The class is created automatically by the runtime or ASP.NET. Some examples of automatically created classes are those that implement <xref:System.Configuration.IConfigurationSectionHandler?displayProperty=fullName> or <xref:System.Web.IHttpHandler?displayProperty=fullName>.
 
-- The class is passed as a type parameter that has a [`new` constraint](../../../csharp/language-reference/keywords/new-constraint.md). The following example will be flagged by rule CA1812:
+- The class is used as a type parameter in a class definition and has a [`new` constraint](../../../csharp/language-reference/keywords/new-constraint.md). The following example will be flagged by rule CA1812:
```

Comment:
Reading the other comments, maybe this clarifies why this example generates the warning:

```suggestion
- The class is used as a type parameter in a class definition and has a [`new` constraint](../../../csharp/language-reference/keywords/new-constraint.md). The generic class instantiates its type parameter, but the isn't instantiated elsewhere. The following example will be flagged by rule CA1812:
```

Reply:
The original issue wasn't about why this case shows the warning, so I'm going to resolve this comment.